tests/cli/test_cli_options.py

```diff
@@ -656,56 +656,60 @@ def main(args):
 
     print("Test help is OK")
 
+    print("Test Version ...")
+    #
+    # Test Version
+    #   -v
+    return_value = check_uncrustify_output(
+            uncr_bin,
+            parsed_args,
+            args_arr=['-v'],
+            out_expected_path=s_path_join(script_dir, 'output/v-out.txt'),
+            out_result_path=s_path_join(test_dir, 'results/v-out.txt'),
+            out_result_manip=reg_replace(r'Uncrustify.+', 'Uncrustify')
+            )
+    # Issue 4379
+    if not return_value:
+        sys_exit(EX_SOFTWARE)
+
+    print("Test Version is OK")
+
+    print("Test help is OK")
```

Comment:
This `print("Test help is OK")` is not needed here, the help test was already done above